resources/js/Pages/Lessons/Show.vue

```diff
@@ -0,0 +1,101 @@
+<script setup>
+import {onMounted, ref} from "vue";
+import unityGameSetUp from '../../GameViewJava';
+import {Head, Link} from '@inertiajs/inertia-vue3';
+
+onMounted(() => {
+    unityGameSetUp();
+    window.addEventListener("stateFromUnity",(e) => recievedJsonFromUnity(e));
+    window.addEventListener("requestData", (e) => requestLoadGameData(e));
+    window.addEventListener("StartChallenge", (e) => recievedChallengeFromUnity(e));
+})
+const emit = defineEmits(['showChallenge'])
+const props = defineProps({
+    lesson: Object ,
+    course: Object,
+
+})
+let showChallenge = ref(false);
+let challengeReponse = ref("hello");
+let challengeAnswer = ref(" ");
+const recievedChallengeFromUnity =  async (e)=>
```

Comment:
```suggestion
})

let showChallenge = ref(false);
let challengeResponse = ref("");
let challengeAnswer = ref("");

const receivedChallengeFromUnity =  async (e) => {
```

Add a blank line before and after the refs this will make things easier to read, the `challengeResponse` and `challengeAnswer` can be an empty string to start or `null`